tools-v2/pkg/cli/command/curvebs/status/copyset/copyset.go

```diff
@@ -135,7 +136,7 @@ func (cCmd *GetCopysetStatusCommand) AddFlags() {
 	config.AddBSPeersConfFlag(cCmd.Cmd)
 }
 
-func GetCopysetStatus(caller *cobra.Command) (*map[string]*copyset.CopysetStatusResponse, *cmderror.CmdError) {
+func GetCopysetStatus(caller *cobra.Command) (map[string]*copyset.CopysetStatusResponse, *cmderror.CmdError) {
```

Comment:
If it returns a value, will it make a copy? If the amount of data is large, it should affect performance more.